tests/dummy/app/pods/view/renderers/documentation/datetime.md

```diff
@@ -3,7 +3,7 @@
 This renders a datetime input.
 
 ### Properties
-
+Currently disabled property is not supported.
```

Comment:
Disabled is a basic functionality bunsen expects. Consumers have cases where they disable forms while saving and in that scenario ALL inputs need to be disabled.